src/components/structures/MessagePanel.js

```diff
@@ -368,6 +369,15 @@ module.exports = React.createClass({
         this.eventNodes[eventId] = node;
     },
 
+    // once dynamic content in the events load, make the scrollPanel check the
+    // scroll offsets.
+    _onWidgetLoad: function() {
+        var scrollPanel = this.refs.scrollPanel;
+        if (scrollPanel) {
+            scrollPanel.checkScroll();
```

Comment:
hm. why doesn't scrollPanel.checkScroll() do this already then?  Especially as scrollPanel's gemini isn't exposed to MessagePanel?